service/apis/entity_service_test.go

```diff
@@ -13,7 +13,6 @@ import (
 
 func TestCreateEntity(t *testing.T) {
 	dc := db.NewDGClient("127.0.0.1:9080")
-	defer dc.Close()
```

Comment:
why removing this line? most of the other tests still seem to have the deferred client close step and I'm not sure where it's happening otherwise in this fn, if not here